ts/store/reducers/notifications/installation.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Notifications installation reducer
+ */
+
+import uuid from "uuid/v4";
+
+import { Action } from "../../../actions/types";
+import { GlobalState } from "../../../reducers/types";
+import { NOTIFICATIONS_INSTALLATION_TOKEN_UPDATE } from "../../actions/constants";
+
+export type InstallationState = Readonly<{
+  uuid: string;
+  token: string | null;
+}>;
+
+export const INITIAL_STATE: InstallationState = {
```

Comment:
this should be a function (e.g. `getInitialState`) or else, `uuid()` will be called every time the app starts, even when `INITIAL_STATE` doesn't get used